README.md

```diff
@@ -1,2 +1,11 @@
-# Hi, I’m BenjammingKirby
-> Discord: [Bwen Jwawwing -3-  🌸🌷💮#8710](https://discord.com/users/754279952647716914)
+# ben
+
+![](https://github.com/BenjammingKirby/BenjammingKirby/raw/output/github-contribution-grid-snake.svg)
+
+## About Me
+
+Hi! My Name is Benjamin Kirby, you can call me Ben! I know what my name sounds like and no, I am not from the 1700s.
```

Comment:
```suggestion
Hi! My Name is Benjamin Kirby, you can call me Ben!
```
After putting that on discord, everyone asks me if i'm from the 1700s, NO, i won't let github users do the same